src/components/NetworkSwitch.tsx

```diff
@@ -0,0 +1,65 @@
+import {
+    Box,
+    Button,
+    DropdownMenuContent,
+    DropdownMenuPortal,
+    DropdownMenuRadioGroup,
+    DropdownMenuRadioItem,
+    DropdownMenuRoot,
+    DropdownMenuTrigger,
+} from "@0xsequence/design-system";
+import { Chain } from "../App.tsx";
+import { useEffect, useState } from "react";
+import { sequence } from "../main.tsx";
+import { ChainListReturn } from "../../../sequence.js/packages/waas/src/clients/authenticator.gen.ts";
```

Comment:
You need to import this from the package, not from the `node_modules` dir. The import should start with `@0xsequence`